atlp-rwanda/e-commerce-lydia-32-fn
# ft-homePageDesign

### What does this PR do?

- This PR implements the homepage design.

### Description of Task to be completed?

- Design the homepage layout and structure.
- Fetch and display products on the homepage.

### How should this be manually tested?

- Start the application.
- Navigate to the homepage.
- Verify that the homepage design is implemented as expected.
- Ensure that products are being fetched and displayed correctly on the homepage.
